<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dCreate</class>
 <widget class="QDialog" name="dCreate">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crear ticket</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widgetDescription" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="lbDescription">
        <property name="text">
         <string>Descripci√≥n</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="tfDescription"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widgetButtons" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btCancel">
        <property name="text">
         <string>Cancelar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btAccept">
        <property name="text">
         <string>Aceptar</string>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
